src/pages/mypage/components/PharmacysCard.tsx

```diff
@@ -2,22 +2,27 @@ import React, { useState } from "react";
 import Image from "next/image";
 import { BookmarkIcon } from "@public/svgs";
 interface PharmacysCardProps {
-  pharmacyName?: string;
-  location?: string;
-  imageUrl?: string;
+  name: string;
+  region: string;
+  img_url: string;
+  latitude: number;
+  longitude: number;
+  place_id: string;
   isBookmarked?: boolean;
   onBookmarkToggle?: () => void;
 }
 
 const PharmacysCard: React.FC<PharmacysCardProps> = ({
-  pharmacyName = "다나아약국",
-  location = "서울 중구 중앙동",
-  imageUrl,
+  name = "다나아약국",
+  region = "서울 중구 중앙동",
```

Comment:
이거 목업 데이터인 것 같은데, 기본값 제거하는게 낫지 않을까요??
```suggestion
  name,
  region,
```

Reply:
넵 삭제했습니다!